src/data/make_dataset.py

```diff
@@ -1,30 +0,0 @@
-# -*- coding: utf-8 -*-
```

Comment:
Why was this file removed?

Reply:
I did not create it. I believe it was automatically generated when I initially created the repo.